# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/boxshadow/box-shadow-large-blur-radius-3.png
# Commit: 15da94c6a7b1
# Full Hash: 15da94c6a7b19f971d3dffaa4cab8edb8e41cb62
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 72ddac486037 -r 15da94c6a7b1 gfx/wr/wrench/reftests/boxshadow/box-shadow-large-blur-radius-3.png
Binary file gfx/wr/wrench/reftests/boxshadow/box-shadow-large-blur-radius-3.png has changed